src/cockpit/beiboot.py

```diff
@@ -260,11 +260,9 @@ class SshBridge(Router):
     ssh_peer: SshPeer
 
     def __init__(self, args: argparse.Namespace):
+        super().__init__([])
         self.ssh_peer = SshPeer(self, args.destination, args)
-
-        super().__init__([
-            DefaultRoutingRule(self, self.ssh_peer),
-        ])
+        self.routing_rules.append(DefaultRoutingRule(self, self.ssh_peer))
```

Comment:
I probably don't understand this fully yet, but this looks weird: `Router.__init__` did not change, it still expects a list of routing rules. I'd prefer to use the official API (which should end up doing the same) instead of poking this into the router's map behind its back. What difference does this change make?